crates/pop-parachains/src/new_parachain.rs

```diff
@@ -120,4 +137,31 @@ mod tests {
 
 		Ok(())
 	}
+
+	#[test]
+	fn test_parachain_instantiate_openzeppelin_template() -> Result<()> {
+		let temp_dir = tempfile::tempdir().expect("Failed to create temp dir");
+		instantiate_openzeppelin_template(&Parachain::OpenZeppelinEVM, temp_dir.path(), None)?;
+
+		// Verify that the generated chain_spec.rs file contains the expected content
+		let generated_file_content =
```

Comment:
Surely this block doesnt apply to this template? Seems as though it has been copied from another test implementation without further consideration.

Reply:
Yes, it was taken from the above and kept it since I don't see any harm, but I agree is an irrelevant test. I removed it.